packages/protocol/test/identity/attestations.ts

```diff
@@ -89,17 +95,62 @@ contract('Attestations', (accounts: string[]) => {
     return accounts[nonIssuerIndex]
   }
 
+  const getValidatingKeyAddress = (address: string) => {
+    const pKey = accountPrivateKeys[accounts.indexOf(address)]
+    const aKey = Buffer.from(pKey.slice(2), 'hex')
+    aKey.write((aKey[0] + 2).toString(16))
+    return privateKeyToAddress('0x' + aKey.toString('hex'))
+  }
+
+  const getAttestationKey = (address: string) => {
+    const pKey = accountPrivateKeys[accounts.indexOf(address)]
+    const aKey = Buffer.from(pKey.slice(2), 'hex')
+    aKey.write((aKey[0] + 1).toString(16))
+    return '0x' + aKey.toString('hex')
+  }
+
+  const unlockAttestationKey = async (address: string) => {
+    const attestationKey = getAttestationKey(address)
+    const attestationKeyAddress = privateKeyToAddress(attestationKey)
+    // @ts-ignore
+    await web3.eth.personal.importRawKey(attestationKey, 'passphrase')
+    await web3.eth.personal.unlockAccount(attestationKeyAddress, 'passphrase', 1000000)
+    return attestationKeyAddress
+  }
+
+  const registerAttestationKey = async (address: string) => {
```

Comment:
nit: `setAttestationKey`? Or `createAndSetAttestationKey`?

Reply:
I named it `setAuthorizedAttestor` along the other renames, wdyt?

Reply:
I like it